main/src/library/File.flix

```diff
@@ -310,9 +310,9 @@ namespace File {
 
             let javaFile = newFile(f);
             let stream = newInputStream(javaFile);
-            skip(stream, opts.offset);
+            discard skip(stream, opts.offset);
```

Comment:
Does this suggest there is a bug? What does skip do? Can it fail?

Reply:
I've made an issue #4006